.flake8

```
@@ -5,7 +5,7 @@ exclude =
     .git,
     __pycache__,
     venv
-ignore = H306, H405
+ignore = H306, H405, H301
```

```suggestion
ignore = H301, H306, H405
```

**H301: Do not import more than one module per line** についてだけど、
[PEP8](https://www.python.org/dev/peps/pep-0008/#imports) に添いましょう。
Hackingは厳格すぎる。

---

そうですね。
`PEP8`で進めます。